cmd/did-method-cli/createconfigcmd/createconfig.go

```diff
@@ -79,10 +78,11 @@ type memberData struct {
 	Policy models.StakeholderSettings `json:"policy"`
 	// Endpoints is a list of sidetree endpoints owned by this stakeholder organization
 	Endpoints []string `json:"endpoints"`
-	// PrivateKeyJwk is privatekey jwk
-	PrivateKeyJwk json.RawMessage `json:"privateKeyJwk,omitempty"`
+	// PrivateKeyJwk is privatekey jwk file
+	PrivateKeyJwkPath string `json:"privateKeyJwkPath,omitempty"`
```

Comment:
this will need some additional followup.

a) the private key might not be in a file.
b) the private key can be protected.
c) might make sense to refactor how keys are associated to stakeholders.

Reply:
will do that as followup